monitor/alerts/model.py

```diff
@@ -1,33 +1,51 @@
 import json
-from dataclasses import dataclass
 from datetime import datetime
-from typing import Union
+from typing import List, Optional, Union
 
 from clients.slack.schema import SlackMessageSchema
 from monitor.alerts.alert import Alert
 from utils.json_utils import prettify_json_str_set
 
 
-@dataclass
 class ModelAlert(Alert):
-    unique_id: str
-    alias: str
-    path: str
-    materialization: str
-    detected_at: Union[str, datetime]
-    database_name: str
-    schema_name: str
-    message: str
-    full_refresh: bool
-    owners: str
-    tags: str
-    status: str
+    def __init__(
```

Comment:
Convert from dataclass into class.
1. IMO dataclass should be used in very specific situations when the class is proper dataclass (and usually it will be better to use pydantic schema)
2. dataclass and inheritance doesn't work so well, and we inheritance (with default values) this class.